tests/semantic_analyser.cpp

```diff
@@ -1892,6 +1892,21 @@ TEST_F(semantic_analyser_btf, short_name)
   test("fr:func_1 { 1 }", 0);
 }
 
+TEST_F(semantic_analyser_btf, call_path)
```

Comment:
The test runs only when HAVE_LIBBPF_BTF_DUMP. So I think it would be better to split this into for kfuncs and for others.

Reply:
ok